vagrant/Brewfile

```diff
@@ -0,0 +1,2 @@
+cask "vagrant"
```

Comment:
You can create a `~/.Brewfile` and use `brew bundle --global` and you can use to install things you always want in your machine, as for example Firefox, Chrome, 1password, etc.